README.md

```diff
@@ -17,6 +17,17 @@ Dogecoin is a community-driven cryptocurrency that was inspired by a Shiba Inu m
 
 **Website:** [dogecoin.com](https://dogecoin.com)
 
+## Dogecoin Policy 📜
```

Comment:
```suggestion
## Default fee policy 📜
```

Reply:
I'm not sure about the right title, but it covers more information than fees

Reply:
It's wise to make a distinction between policies and consensus rules:

Policies are just recommendations that relay nodes, miners and wallets can choose to ignore. Consensus rules such as block timing and subsidy cannot be changed without a fork and aren't recommendations at all.

Reply:
Any term in mind to cover both ?

Reply:
Hmm how about "About the Dogecoin blockchain" as a level 2 header, and then we can have a subsection "Recommended fee policies"? That'd mean doing something like:

```

## About the Dogecoin blockchain

 | Parameter         | Value       |
 | :---------------- | -----------:|
 | Block time        |    1 minute |
 | Block reward      |    10'000 Ð |
 | Hashing algorithm |      scrypt |

### Recommended fee policies

The values below are the developer-recommended defaults when creating and processing transactions:

 | Parameter       | Value       |
 | :-------------- | -----------:|
 | Transaction fee |   0.01 Ð/kb |
 | Minimum output  |      1.02 Ð |
 | RBF increment   |     0.001 Ð |

```

Reply:
For the documentation, personally I do not really like this idea of having 2 arrays, I find it really verbose presented like that.

What about it ?
 | Parameter       | Value       |
 | :-------------- | -----------:|
 | Block time        |    1 minute |
 | Block reward      |    10'000 Ð |
 | Hashing algorithm |      scrypt |
 | Recommended Transaction fee |   0.01 Ð/kb |
 | Recommended Minimum output  |      1.02 Ð |
 | Recommended RBF increment   |     0.001 Ð |